core/src/util/ext-tools.ts

```diff
@@ -71,9 +72,6 @@ export abstract class CliWrapper {
     if (!args) {
       args = []
     }
-    if (!cwd) {
-      cwd = dirname(path)
-    }
 
     log.silly(() => `Execing '${path} ${args!.join(" ")}' in ${cwd}`)
```

Comment:
Let's log the actual CWD, instead of `undefined`
```suggestion
    log.silly(() => `Execing '${path} ${args!.join(" ")}' in ${cwd | resolve('.'}`)
```